src/graph/validator/MatchValidator.cpp

```diff
@@ -818,9 +823,16 @@ Status MatchValidator::validateGroup(YieldClauseContext &yieldCtx,
 
         yieldCtx.needGenProject_ = true;
         yieldCtx.aggOutputColumnNames_.emplace_back(agg->toString());
+
+        for (auto *labelExpr : labelExprs) {
+          yieldCtx.groupKeys_.emplace_back(labelExpr->clone());
```

Comment:
`match (v) return {abs(v.age),count(v.name)+1}`
You should groupby `abs(v.age)` rather than `v.age`.